crates/sui-indexer/src/store/pg_partition_manager.rs

```diff
@@ -116,6 +131,68 @@ impl<T: R2D2Connection> PgPartitionManager<T> {
         )
     }
 
+    /// Tries to fetch the partitioning strategy for the given partitioned table. Defaults to
+    /// `CheckpointSequenceNumber` as the majority of our tables are partitioned on an epoch's
+    /// checkpoints today.
+    pub fn get_strategy(&self, table_name: &str) -> &PgPartitionStrategy {
```

Comment:
To go with the suggestion to derive `Copy` on this type.

```suggestion
    pub fn get_strategy(&self, table_name: &str) -> PgPartitionStrategy {
```